src/main/java/com/comcast/zucchini/AbstractZucchiniTest.java

```diff
@@ -86,21 +140,25 @@ public void runParallel(List<TestContext> contexts) {
             Thread t = new Thread(new TestRunner(this, tc, mi), tc.name);
             threads.add(t);
             t.start();
-            //threads.push(Thread.start(new TestRunner(this, tc, mi), tc.name));
         }
 
         for(Thread t : threads) {
             try {
                 t.join();
             }
-            catch(Throwable e) {
-                logger.error(t.toString());
+            catch(InterruptedException e) {
+                LOGGER.error("ERROR on Thread.join(): {}", e);
```

Comment:
That might be a good idea, thought the fail should wait until they have all joined or failed, if we choose to go that route.  We might need to add the MojoExecutionException back in as the 'throw' if we want it run later tests.  

On a side note, I don't think that the InterruptedException can make it out to this point unless the interrupted flag is set and is then allowed to propagate.  If it is raised in on the of the step definitions, it won't make it back up to this level.  I'll have to check the docs again to be sure.